kt/plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/tasks/GenerateEmbeddedJreTask.kt

```diff
@@ -0,0 +1,62 @@
+package godot.gradle.tasks
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+import java.util.*
+
+open class GenerateEmbeddedJreTask : DefaultTask() {
+
+    @Input
+    var modules: Array<String> = arrayOf(
+        "java.base",
+        "java.logging",
+    )
+
+    @Input
+    var outputDir: String = "jvm/jre-${getArch()}-${getOs()}"
+
+    @Input
+    var arguments: Array<String> = arrayOf(
+        "--strip-debug",
+        "--no-header-files",
+        "--no-man-pages"
+    )
+
+    @Input
+    var javaHome: String = System.getProperty("java.home")
```

Comment:
Can we potentially use the JVM running the gradle task if JAVA_HOME is not set ? Also can we make sure it's at least 11 ?